src/libproxychains.c

```diff
@@ -290,7 +303,7 @@ static void get_chain_data(proxy_data * pd, unsigned int *proxy_count, chain_typ
 	if( ( file = fopen(env, "r") ) == NULL )
 	{
 	        perror("couldnt read configuration file");
-        	exit(1);
+	    	exit(1);
```

Comment:
pls dont change indentation in unrelated code

Reply:
fixed in latest commit.